miden-crypto/src/lib.rs

```diff
@@ -30,36 +30,37 @@ pub use word::{Word, WordError};
 
 /// An alias for a key-value map.
 ///
-/// By default, this is an alias for the [`alloc::collections::BTreeMap`], however, when the
-/// `hashmaps` feature is enabled, this is an alias for the `hashbrown`'s `HashMap`.
-#[cfg(feature = "hashmaps")]
-pub type Map<K, V> = hashbrown::HashMap<K, V>;
+/// By default, this is an alias for the [`alloc::collections::BTreeMap`]; when the `std` feature is
+/// enabled, this is an alias for the `std::collections::HashMap` (iteration order is not
+/// deterministic in that case).
+#[cfg(feature = "std")]
+pub type Map<K, V> = std::collections::HashMap<K, V>;
 
-#[cfg(feature = "hashmaps")]
-pub use hashbrown::hash_map::Entry as MapEntry;
+#[cfg(feature = "std")]
+pub use std::collections::hash_map::Entry as MapEntry;
 
 /// An alias for a key-value map.
 ///
 /// By default, this is an alias for the [`alloc::collections::BTreeMap`], however, when the
-/// `hashmaps` feature is enabled, this is an alias for the `hashbrown`'s `HashMap`.
-#[cfg(not(feature = "hashmaps"))]
+/// `std` feature is enabled, this is an alias for the `std::collections::HashMap`.
+#[cfg(not(feature = "std"))]
 pub type Map<K, V> = alloc::collections::BTreeMap<K, V>;
```

Comment:
Same comment as above: by default we enable `std::collections::HashMap`.

This is also relevant in a couple other places in this file.